dep/wire.go

```diff
@@ -77,6 +77,7 @@ func InitGRpcService(
 	sendGridAPIKey provider.SendGridAPIKey,
 	templateRootDir provider.TemplateRootDir,
 	cacheSize provider.CacheSize,
+	eventDispatcher fw.Dispatcher,
```

Comment:
I wonder whether this is to ensure we use a global dispatcher?

Reply:
I think it is a justified decision. We could have different processes running besides `GRPC service`:
* monitoring metrics sender
* queue message handler

What do you think?

Reply:
Sounds great!